sandbox/my_sod_shock_stube_solver/my_cese_solver.py

```diff
@@ -0,0 +1,186 @@
+"""
+Sod Shock Tube Problem -- CESE solver
+Editor: You-Hao Chang
```

Comment:
Usually author means the person who writes this code.  I guess editor is not often used in authorship declaration.